### ISSUE TITLE
[Feature Request]display image along with labels

### ISSUE THREAD

**Issue by qianyizhang** [DIYer22/boxx#10]
have you implement anything that displays image long with its predictions? (bbox, mask, heat map, etc.)
also it's best if the input can be torch.tensors

**Comment by DIYer22** [comment]
For mask and heatmap, I write several lines of code per case to visualize.    
For bbox, I use [detectron's vis code](https://github.com/facebookresearch/Detectron/blob/master/detectron/utils/vis.py)

**Comment by qianyizhang** [comment]
thanks, i was hoping for some more generalized api, but this also works
